iphone/Classes/TiUIiPhoneSystemIconProxy.m

```diff
@@ -16,19 +16,18 @@ -(NSString*)apiName
     return @"Ti.UI.iPhone.SystemIcon";
 }
 
-MAKE_SYSTEM_PROP(BOOKMARKS,UITabBarSystemItemBookmarks);
-MAKE_SYSTEM_PROP(CONTACTS,UITabBarSystemItemContacts);
-MAKE_SYSTEM_PROP(DOWNLOADS,UITabBarSystemItemDownloads);
-MAKE_SYSTEM_PROP(FAVORITES,UITabBarSystemItemFavorites);
-MAKE_SYSTEM_PROP(FEATURED,UITabBarSystemItemFeatured);
-MAKE_SYSTEM_PROP(HISTORY,UITabBarSystemItemHistory);
-MAKE_SYSTEM_PROP(MORE,UITabBarSystemItemMore);
-MAKE_SYSTEM_PROP(MOST_RECENT,UITabBarSystemItemMostRecent);
-MAKE_SYSTEM_PROP(MOST_VIEWED,UITabBarSystemItemMostViewed);
-MAKE_SYSTEM_PROP(RECENTS,UITabBarSystemItemRecents);
-MAKE_SYSTEM_PROP(SEARCH,UITabBarSystemItemSearch);
-MAKE_SYSTEM_PROP(TOP_RATED,UITabBarSystemItemTopRated);
-
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(BOOKMARKS,  UITabBarSystemItemBookmarks,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(CONTACTS,   UITabBarSystemItemContacts,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(DOWNLOADS,  UITabBarSystemItemDownloads,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(FAVORITES,  UITabBarSystemItemFavorites,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(FEATURED,   UITabBarSystemItemFeatured,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(HISTORY,    UITabBarSystemItemHistory,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(MORE,       UITabBarSystemItemMore,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(MOST_RECENT,UITabBarSystemItemMostRecent,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(MOST_VIEWED,UITabBarSystemItemMostViewed,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(RECENTS,    UITabBarSystemItemRecents,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(SEARCH,     UITabBarSystemItemSearch,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
+MAKE_SYSTEM_PROP_DEPRECATED_REPLACED(TOP_RATED,  UITabBarSystemItemTopRated,@"UI.iPhone.SystemIcon.BOOKMARKS",@"6.0.0",@"UI.iOS.SystemIcon.BOOKMARKS");
 
```

Comment:
All properties wrong..**Don't copy and paste!!**. Also fix the indentation level.